src/translations/screens/Profile.ts

```diff
@@ -352,6 +352,14 @@ const ProfileTexts = {
       'Klikk for å kopiere id. Denne blir brukt til teknisk feilsøk.',
     ),
   },
+  orgNumber: (orgNr: string) =>
+    _(`Org.nr: ${orgNr}`, `Org.nr: ${orgNr}`, `Org.nr: ${orgNr}`),
+  orgNumberA11yLabel: (orgNr: string) =>
+    _(
+      `Organisasjonsnummer ${orgNr}`,
+      `Organization number ${orgNr}`,
```

Comment:
Did we standardise the use of British English yet?
Should be `Organisation Number` I think